app/models/concerns/file_backed_model.rb

```diff
@@ -51,16 +51,11 @@ def from_file(filename)
     # Returns a Pathname to location configured in the database through the
     # :setting_name setting
```

Comment:
Due to the refactor, this comment is not relevant anymore and can be removed

Reply:
fixed.